packages/paste-website/src/pages/getting-started/engineering/index.mdx

```diff
@@ -49,7 +49,33 @@ export const pageQuery = graphql`
 
 <content>
 
-## Installation
+## Quickstart
+
+The quickest way to get started with Paste is to bootstrap a new React app using one of our templates for popular React frameworks, [Next.js](https://nextjs.org/) and [Create React App](https://create-react-app.dev/).
+
+Both approaches will create a new React app with Paste installed and set up ready to start developing with.
```

Comment:
`with Paste installed and ready for development.` maybe?

Reply:
I dig it